packaging/MacSDK/msbuild.py

```diff
@@ -7,6 +7,7 @@ def __init__ (self):
 
 	def build (self):
 		self.sh ('./build.sh -host mono -configuration Release -skipTests')
+		self.sh ('zip msbuild-bin-logs.zip artifacts/Release-MONO/log/*')
```

Comment:
@akoeplinger @alexischr  How do I get the path for `mono_repo` here, short a hardcoded relative path?